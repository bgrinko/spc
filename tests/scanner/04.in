string '12345344334'