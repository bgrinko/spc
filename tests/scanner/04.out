1	6	NONE		string
1	21	NONE		12345344334
